.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: 'release'
+on:
+  workflow_dispatch:
```

Comment:
can we configure it to run weekly automatically?

Reply:
I will see into that